src/commands/test.js

```diff
@@ -121,16 +121,18 @@ async function runBinary(datasource, opts) {
   args.length > 0 ? binary.run(...args) : binary.run()
 }
 
-function getPlatform(logsOpt) {
+async function getPlatform(logsOpt) {
   const type = os.type()
   const arch = os.arch()
-  const release = os.release()
   const cpuCore = os.cpus()[0]
-  const majorVersion = semver.major(release)
   const isM1 = cpuCore.model.includes("Apple M1")
+  const linuxInfo = type === 'Linux' ? await getLinuxInfo() : []
+  const linuxDestro = linuxInfo[0] || type
+  const release = linuxInfo[1] || os.release()
+  const majorVersion = parseInt(linuxInfo[1]) || semver.major(release)
 
   if (logsOpt) {
-    print.info(`OS type: ${type}\nOS arch: ${arch}\nOS release: ${release}\nOS major version: ${majorVersion}\nCPU model: ${cpuCore.model}`)
+    print.info(`OS: ${linuxDestro || type}\nOS arch: ${arch}\nOS release: ${release}\nOS major version: ${majorVersion}\nCPU model: ${cpuCore.model}`)
```

Comment:
I don't want to force a refactor into the function, but maybe the `||` here could be in the value `type` itself, but as mentioned, this would require moving somethings around, Iet that decision to you 🙂

Reply:
`type` is used later and I don't want to overwrite it with the linux distro name, although the `const linuxDistro = linuxInfo[0] || type` currently does not make sense atm